backend/src/routes/events.ts

```diff
@@ -185,7 +185,7 @@ async function listAndWatch(options: IWatchOptions) {
 }
 
 async function listKubernetesObjects(options: IWatchOptions) {
-    const serviceAcccountToken = getServiceAcccountToken()
+    const serviceAcccountToken = getServiceAccountToken()
```

Comment:
The const var name still has the extra "c" `const serviceAcccountToken`

Reply:
Fixed